File: conditionals/time_of_day.py
# Lila Shearer, Time of Day Python

import time
current_time = time.localtime()  
hour = current_time.tm_hour  

if hour > 0 and hour < 12:
    print("Good morning!\n")
elif hour >12 and hour <17:
    print("Good afternoon!\n")
else:
    print("Good night!\n")

File: conditionals/time_of_day.c
// Lila Shearer, Time of Day C
#include <stdio.h>
#include <time.h>  

int main(void){
    time_t t = time(NULL);
    struct tm *tm = localtime(&t);
    int hour = tm->tm_hour;
    if (hour > 0 && hour < 12){
    printf("Good morning!\n");
    }else if (hour >12 && hour <17){
    printf("Good afternoon!\n");
    }else{
    printf("Good night!\n");
    }
    return 0;
}